doc/lua_api.txt

```diff
@@ -6922,10 +6922,11 @@ Used by `minetest.register_node`.
             -- Node was placed. Also played after falling
 
             place_failed = <SimpleSoundSpec>,
-            -- When node placement failed
+            -- When node placement failed.
+            -- Note: Does not take `on_place` into account
```

Comment:
The sound does not take `on_place` into account?
What?

Reply:
@Wuzzy2 This description is unclear. Did you mean something like this?
`Note: Is not played when 'on_place' is overridden`